pkgs/applications/networking/sync/rclone/default.nix

```
@@ -41,6 +41,11 @@ buildGoModule rec {
         ${rcloneBin}/bin/rclone genautocomplete $shell rclone.$shell
         installShellCompletion rclone.$shell
       done
+
+      # filesystem helpers
+      ln -sf $out/bin/rclone $out/bin/rclonefs
+      mkdir -p $out/sbin
+      ln -sf $out/bin/rclone $out/sbin/mount.rclone
```

Does this need to be sbin? NixOS does not make this difference and other mount helpers are also not under sbin


```suggestion
      ln -sf $out/bin/rclone $out/bin/mount.rclone
```

---

It does not but i think I saw other `mount.something` put it in `sbin` so I did the same to keep the convention.

---

Do you want me to change it?

---

does it work without the `sbin` symlink? if so please change it

---

I tried to do it, but there's way too many references to `sbin` everywhere in those packages. I don't feel confident changing their code.

I'll just do for `rclonefs`